package.json

```diff
@@ -49,28 +48,23 @@
     "@nomicfoundation/hardhat-chai-matchers": "^2.0.6",
     "@nomicfoundation/hardhat-ethers": "^3.0.5",
     "@nomicfoundation/hardhat-network-helpers": "^1.0.11",
+    "@openzeppelin/contracts": "file:lib/@openzeppelin-contracts",
```

Comment:
As a side effect, this fill get all the dependencies that are already present in the main repo.

That means we could technically remove anything we have in `devDependencies` that is not exclusive to the community repo. Do we want to do that ?